internal/project/dirty/interfaces.go

```diff
@@ -0,0 +1,21 @@
+package dirty
+
+type cloneable struct{}
```

Comment:
Is this used anywhere? Or is it just for the type assertion?

Reply:
It’s just used for the assertion 😕 I couldn’t find another way to do this; any suggestions? I can just delete the assertions.

Reply:
clonableDoNotUseOrYouWillBeFired